src/celttf/truetypefont.h

```diff
@@ -27,7 +27,7 @@ class TextureFont
     TextureFont& operator=(const TextureFont&) = delete;
     TextureFont& operator=(TextureFont&&) = delete;
 
-    void setMVPMatrix(const Eigen::Matrix4f& m);
+    void setMVPMatrices(const Matrices &mvp);
```

Comment:
not sure whether `Matrices` is good here, i see there is a lot places where you use `{&m, &p}` maybe `Matrix4f &m, Matrix4f &p` is better?

Reply:
done, prefer projection then model view.